### PR TITLE
Add missing LTI field

### PR DESCRIPTION
**Pull request by mhoeber**
@lamagnifica

### CODE REVIEW

File: en_us/shared/exercises_tools/lti_component.rst [hunk 1]
```diff
@@ -216,6 +216,10 @@ LTI Component Settings
        component as a placeholder for syncing with an external grading system.
        If you set the value to True, Studio hides the **Launch** button and any
        IFrames for this component. By default, this value is set to False.
+   * - LTI Application Information     
+     - The description of the third party application. If the application
+       requires a username or email, use this field to inform learners why
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/lti_component.rst]: email ^address (?)

**Comment by lamagnifica** [comment]
:+1: (I prefer email address to just email, but that may just be me...)

File: en_us/shared/exercises_tools/lti_component.rst [hunk 2]
```diff
@@ -216,6 +216,10 @@ LTI Component Settings
...
+       their username or email will be forwarded to a third party application.
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/lti_component.rst]: ditto

**Comment by mhoeber** [comment]
updated with address

**Pull request merged**